runtime/src/main/kotlin/kotlin/native/ForeignException.kt

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright 2010-2018 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license
+ * that can be found in the LICENSE file.
+ */
+package kotlin.native
+
+import kotlin.RuntimeException
+import kotlin.native.internal.NativePtr
+
+public class ForeignException internal constructor(val objCException: Any?)  : RuntimeException()
```

Comment:
Can `objCException` actually be `null`?

Reply:
There is no guarantee: think of `throw (NSException*)nullptr;` in user code.

Reply:
I see. Makes sense.